core/CronArchive.php

```diff
@@ -1760,6 +1772,18 @@ private function makeRequestUrl($url)
         return $url;
     }
 
+    protected function wasSegmentCreatedRecently($definition, $allSegments)
+    {
+        foreach ($allSegments as $segment) {
+            if ($segment['definition'] === $definition) {
+                $twentyFourHoursAgo = Date::now()->subHour(24);
+                return Date::factory($segment['ts_created'])->isLater($twentyFourHoursAgo);
```

Comment:
we use `ts_created` but could we instead use `ts_last_edit`? For the use case where user is updating the segment after a few days, and would also expect the segment to be processed soon after the modification (ideally).

Reply:
updated the PR to look at that also